nengo_gui/static/status_bar.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Simple JavaScript status bar which is used at the bottom of the net graph
+ * window.
+ *
+ * Creates a global instance Nengo.status_bar inside the netgraph which can be
+ * accessed using the following functions:
+ * - Nengo.status_bar.set_caption()
+ * - Nengo.status_bar.get_caption()
+ * - Nengo.status_bar.hide()
+ * - Nengo.status_bar.show()
+ *
+ * @author Andreas Stöckel
+ * @date November 2017
```

Comment:
We usually do not include authorship and date information in the source files. The VCS takes care of that information more accurately anyways.

Reply:
Thanks, changed that.